### ISSUE TITLE
Needinfo on inactive bugs with assignee and no needinfo

### ISSUE THREAD

**Issue by marco-c** [mozilla/relman-auto-nag#1289]
E.g. [bug 1732421](https://bugzilla.mozilla.org/show_bug.cgi?id=1732421) or [bug 1684212](https://bugzilla.mozilla.org/show_bug.cgi?id=1684212) (has open dep, no needinfo)

**Comment by marco-c** [comment]
The first step should be to investigate how noisy this would be and if we can limit it in some way to make it useful but not noisy.

**Comment by tnikkel** [comment]
If someone is working on something else that is a higher priority polluting their inbox with useless needinfos is just going to put them further behind.

**Comment by marco-c** [comment]
Adding "useless" needinfos is definitely not our goal. *If* we are going to do this, the first step is what I mentioned above: "investigate how noisy this would be and if we can limit it in some way to make it useful but not noisy.".

**Comment by tnikkel** [comment]
I'm saying that all of these needinfos aren't useful. So like the investigation is done?

**Comment by marco-c** [comment]
Sorry, I'm not sure I get what you mean. Could you rephrase or be more verbose?

**Comment by tnikkel** [comment]
You are saying that we need to investigate to determine the trade off between noiseyness and these needinfos. I'm saying the investigation is done because the needinfos aren't useful, so there is no trade off to be made.

**Comment by marco-c** [comment]
No investigation was done yet as far as I know. Investigating means writing the queries, checking how many bugs would be touched, checking that they are not too many, ensuring that the percentage of bugs where the action is meaningful is high enough, adjusting the queries, etc., etc..

This tool was suggested by decoder as during weekly regression triage the release managers or the regression engineering owner needinfo people on such bugs.
See for example https://bugzilla.mozilla.org/show_bug.cgi?id=1732421#c7:
![immagine](https://user-images.githubusercontent.com/1616846/196378916-6160c5f8-864e-4235-bcec-1f5d305fef25.png)

Or https://bugzilla.mozilla.org/show_bug.cgi?id=1684212#c2:
![immagine](https://user-images.githubusercontent.com/1616846/196379183-54e6d00d-b0d2-4386-89d6-8cf4ef57b99d.png)

The idea here is **not** to add more needinfos, it's to automate the needinfos that currently release managers or regression engineering owners are already setting. The net result should be 0 new needinfos.
